### PR TITLE
the draw method -> the `draw()` method

### PR DESCRIPTION
**Pull request by NG-Glen**

### CODE REVIEW

File: tutorials/circuits_advanced/03_advanced_circuit_visualization.ipynb [hunk 1]
```diff
@@ -178,7 +178,7 @@
...
+    "By default, the `draw()` method returns the rendered image as an object and does not output anything. The exact class returned depends on the output specified: `'text'`(the default) returns a `TextDrawer` object, `'mpl'` returns a `matplotlib.Figure` object, and `latex` returns a `PIL.Image` object. Having the return types enables modifying or directly interacting with the rendered output from the drawers. Jupyter notebooks understand these return types and render them for us in this tutorial, but when running outside of Jupyter, you do not have this feature automatically. However, the `draw()` method has optional arguments to display or save the output. When specified, the `filename` kwarg takes a path to which it saves the rendered output. Alternatively, if you're using the `mpl` or `latex` outputs, you can leverage the `interactive` kwarg to open the image in a new window (this will not always work from within a notebook but will be demonstrated anyway)."
```
[the remark this answers is not in the document]
Comment (NG-Glen) [reply tutorials/circuits_advanced/03_advanced_circuit_visualization.ipynb:181]: In this line have 2 part to talk about "draw() method", first is about returns, another for arguments, "the draw() method" wording keep consistent

[the remark this answers is not in the document]
Comment (garrison) [reply tutorials/circuits_advanced/03_advanced_circuit_visualization.ipynb:181]: My suggestion adds a space to a different part of the line.

[the remark this answers is not in the document]
Comment (NG-Glen) [reply tutorials/circuits_advanced/03_advanced_circuit_visualization.ipynb:181]: I'm working with translate Qiskkit to other language, it's more like key word, and don't meed to translate. If use draw(), any translator will be understand it.

**Review by HuangJunye — APPROVED** [review]
Thanks for your contributions!

**Pull request merged**